### DISCUSSION TITLE
Email ID is not provided by our college

### DISCUSSION THREAD

**Discussion by Phani4658** [community/community#63667] [GitHub Education]
Hello there, 

This is to bring to your notice that our college doesn't provide email ID's for the batch of 2021 - 2025. I have been using Student developer pack without any issues from past 2 years even without school email. It is not fair to ask for email ID now. Currently i am in my second year and actively using the Github pro for my projects. Please look into this issue as soon as possible.

**Comment by Prakash4844** [comment]
Hey, i think you can still get student developers pack by verifying your college ID with GitHub.

**Reply by Prakash4844** [reply]
Hey as you can see. it is clearly listed that college issued documents can be used to verify student status.

![image](https://github.com/community/community/assets/81550376/f59920ef-387a-4202-94ca-70aa47682b57)


Did you provide a clear image of your College ID While applying for Student developers pack?

Open This Link: https://education.github.com/discount_requests/application

See what it says in Right hand panel. it might provide some details.

**Reply by Prakash4844** [reply]
actually when i upload the id, i provided both Front and Back images of the ID, maybe try that.

please do not post your ID publicly.

if nothing works, then github support is only option.

**Reply by c-ryan-z** [reply]
Hey @Prakash4844, where can I find the option to upload school documents instead of a school-issued email? I can't seem to find it anywhere.

**Comment by Prakash4844** [comment]
Hey i can see your account already have PRO tag?

**Reply by Phani4658** [reply]
My Developer pack is gonna expire in 4 days

**Comment by DSDmark** [comment]
Hi @Phani4658,

:warning: Please don't upload your personal information or any documents here.
